<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterGenerator</class>
 <widget class="QWidget" name="CharacterGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="generateCharacterButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Character</string>
   </property>
  </widget>
  <widget class="QComboBox" name="classComboBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>&lt;Choose one&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fighter</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="rollAbilityScoresButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Roll Ability Scores</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="raceComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>&lt;Choose one&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Human</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="playerPicture">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="characterName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>28</number>
   </property>
  </widget>
  <widget class="QLabel" name="classLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Class:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="genderComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>&lt;Choose one&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="raceLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Race:</string>
   </property>
  </widget>
  <widget class="QLabel" name="raceLabel_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Character Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnToMenuButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Return to menu</string>
   </property>
  </widget>
  <widget class="QListWidget" name="dicerollList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>91</width>
     <height>111</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="raceLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveCharacterButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Character</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/characterCreationBackground.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="typeComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>&lt;Choose one&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Brute</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nimble</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tank</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="raceLabel_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>generateCharacterButton</zorder>
  <zorder>classComboBox</zorder>
  <zorder>rollAbilityScoresButton</zorder>
  <zorder>raceComboBox</zorder>
  <zorder>playerPicture</zorder>
  <zorder>characterName</zorder>
  <zorder>classLabel</zorder>
  <zorder>genderComboBox</zorder>
  <zorder>raceLabel</zorder>
  <zorder>raceLabel_3</zorder>
  <zorder>returnToMenuButton</zorder>
  <zorder>dicerollList</zorder>
  <zorder>raceLabel_2</zorder>
  <zorder>saveCharacterButton</zorder>
  <zorder>typeComboBox</zorder>
  <zorder>raceLabel_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
